HW0/ans/P6.txt

```diff
@@ -0,0 +1,7 @@
+A. (1) By coding it for several times, the results are different each time. The results printed can be grouped every four items. In each group, the set of the four items are the same each time while the order of the four items are random.
```

Comment:
I am not sure if it is strictly within these 4 groups of 4 items. It might be theoretically possible for Task 5 to run before Task 4, depending on how the threads are managed. This may happen if the tasks are much longer in duration